libmamba/src/download/curl.cpp

```diff
@@ -28,6 +29,9 @@ namespace mamba
             const std::string& ssl_verify
         )
         {
+            std::ofstream out("trace.txt", std::ios_base::app);
+            out << "configuring CURL handle:"
+                << "\n\turl = " << url << std::endl;
```

Comment:
What is this trace.txt?

Reply:
A leftover, I need to remove them, I just wanted to get all the tests green before removing it.